src/App.tsx

```diff
@@ -77,13 +71,15 @@ const App: FC = () => {
           dispatch(setNotification(feideUser.error, NotificationVariant.Error));
           setIsLoadingUser(false);
         } else {
-          if (feideUser.cristinId) {
+          if (feideUser['custom:cristinId']) {
             dispatch(setUser(feideUser));
-          } else {
+          } else if (feideUser['custom:customerId']) {
             const cristinId = await getCustomerInstitution(feideUser['custom:customerId']).then(
```

Comment:
hva er plana med denne? skal den bare få leve som en feilsikring - eller skal den bort en dag?

Reply:
jeg bare tenkte å la den ligge sånn i tilfelle vi kun får customerId, men kan fjerne den også da. Hva tror du?

Reply:
har egentlig mest lyst til å bare fjerne, så kommer det bare enda bedre frem om vi har problemer med dette i fremtiden også (for det er vel en backendfeil som bør jo fikses asap?) - men jeg kan leve med at det ligger der også